src/Build/BackEnd/Components/Communications/NodeProviderOutOfProc.cs

```diff
@@ -72,17 +72,22 @@ public int AvailableNodes
         /// Derived from the binary timestamp to avoid mixing binary versions,
         /// Is64BitProcess to avoid mixing bitness, and enableNodeReuse to
         /// ensure that a /nr:false build doesn't reuse clients left over from
-        /// a prior /nr:true build.
+        /// a prior /nr:true build. The enableLowPriority flag is to ensure that
+        /// a build with /low:false doesn't reuse clients left over for a prior
+        /// /low:true build.
         /// </summary>
         /// <param name="enableNodeReuse">Is reuse of build nodes allowed?</param>
-        internal static long GetHostHandshake(bool enableNodeReuse)
+        /// <param name="enableLowPriority">Is the build running at low priority?</param>
+        internal static long GetHostHandshake(bool enableNodeReuse, bool enableLowPriority)
         {
             long baseHandshake = Constants.AssemblyTimestamp;
 
             baseHandshake = baseHandshake*17 + EnvironmentUtilities.Is64BitProcess.GetHashCode();
```

Comment:
Aside because it's not changed by your PR, but I hadn't noticed that the hashing scheme was now mixing all the bits. There is of course the possibility, however small, that that could cause a match when there should not be. Consider eg., if we changed the impolementation of string.GetHashCode() to return all zeros - that's absurd but valid. I thought originally this method was setting specific bits for specific features. Eg., 64 bitness could toggle one bit. Not sure why it's mixing the bits now.